config/routes.rb

```diff
@@ -4,7 +4,12 @@
     get 'dashboard', to: 'dashboards#index', as: 'dashboard'
     get 'reports', to: 'reports#index', as: 'reports'
     post 'search', to: 'dashboards#search', defaults: { format: :js }
-    resources :users, only: [:new, :create, :index]
+
+    post 'users/search', to: 'users#search', defaults: { format: :js }
+    get 'users/reset_password/:user_id', to: 'users#reset_password', as: 'reset_password'
```

Comment:
So this is a little more rest-y, can you swap the placement of reset_password and :user_id here? So have it end up as `users/:user_id/reset_password`, and below, `users/:user_id/toggle_lock`.